matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/state/SendStateTask.kt

```diff
@@ -39,7 +39,7 @@ internal class DefaultSendStateTask @Inject constructor(
 
     override suspend fun execute(params: SendStateTask.Params) {
         return executeRequest(globalErrorReceiver) {
-            if (params.stateKey == null) {
+            if (params.stateKey.isEmpty()) {
```

Comment:
Why do we even need to test this?

Reply:
probably if 
`/_matrix/client/r0/rooms/{roomId}/state/{eventType}/` is not equivalent to
`/_matrix/client/r0/rooms/{roomId}/state/{eventType}`